### ISSUE TITLE
Ubuntu screen reader does not read "getting started" descriptions

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#229612]
Testing #228766

1. Enable the default screenreader in Ubuntu and editor accessibility mode
2. Focus on a getting started item (like this one.) Only the bold heading is read, I can't figure out how to get it to read the description
![Image](https://github.com/user-attachments/assets/5c9b08db-281d-4468-87fe-851730671781)

**Comment by meganrogge** [comment]
@connor4312 I think this is probably because you don't know how to navigate content using that screen reader? In any case, you should be able to read it in the accessible view.

**Comment by connor4312** [comment]
Probably! Yes the accessible view worked
